po/ro.po

```diff
@@ -0,0 +1,703 @@
+
+#: src/clean.rs:93
+msgid "can't open clone dir: {}"
+msgstr "nu se poate deschide directorul clone: {}"
+
+#: src/command_line.rs:127
+msgid "option {} expects a value"
+msgstr "opțiunea {} așteaptă o valoare"
+
+#: src/command_line.rs:158 src/command_line.rs:327
+msgid "option {} does not allow a value"
+msgstr "opțiunea {} nu permite alocarea unei valori"
+
+#: src/command_line.rs:318
+msgid "unknown option --{}"
+msgstr "opțiune necunoscută --{}"
+
+#: src/command_line.rs:321
+msgid "unknown option -{}"
+msgstr "opțiune necunoscută -{}"
+
+#: src/completion.rs:27
+msgid "failed to open cache file '{}'"
+msgstr "nu s-a putut deschide fișierul cache '{}'"
+
+#: src/completion.rs:65
+msgid "could not update aur cache"
+msgstr "nu s-a putut actualiza cache-ul aur"
+
+#: src/config.rs:174
+msgid "invalid value '{val}' for key '{key}', expected: {exp}"
+msgstr ""
+"valoare incorectă '{val}' pentru cheia '{key}', valoare așteptată: {exp}"
+
+#: src/config.rs:474
+msgid "failed to find cache directory"
+msgstr "nu s-a găsit directorul cache"
+
+#: src/config.rs:477
+msgid "failed to find config directory"
+msgstr "nu s-a găsit directorul config"
+
+#: src/config.rs:501
+msgid "config file '{}' does not exist"
+msgstr "fișierul de configurare '{}' nu există"
+
+#: src/config.rs:550
+msgid "only one operation may be used at a time"
+msgstr "se poate folosi doar o singură operațiune la un anumit moment"
+
+#: src/config.rs:638
+msgid "can not find local repo '{}' in pacman.conf"
+msgstr "nu s-a putut găsi repo-ul local '{}' în pacman.conf"
+
+#: src/config.rs:670
+msgid "failed to initialize alpm: root={} dbpath={}"
+msgstr "nu s-a putut inițializa alpm: root={} dbpath{}"
+
+#: src/config.rs:740 src/config.rs:873
+msgid "value can not be empty for key '{}'"
+msgstr "valoarea cheii '{}' nu poate sa fie empty"
+
+#: src/config.rs:756
+msgid "key '{}' does not belong to a section"
+msgstr "cheia '{}' nu aparține unei secțiuni"
+
+#: src/config.rs:762
+msgid "unknown section '{}'"
+msgstr "secțiune necunoscută '{}'"
+
+#: src/config.rs:769 src/config.rs:824 src/config.rs:829
+msgid "key can not be empty"
+msgstr "cheia nu poate să fie empty"
+
+#: src/config.rs:791
+msgid "error: unknown option '{}' in section [bin]"
+msgstr "eroare: opțiune necunoscută '{}' în secțiunea [bin]"
+
+#: src/config.rs:891
+msgid "error: unknown option '{}' in section [options]"
+msgstr "erorare: opțiune necunoscută '{}' în secțiunea [options]"
+
+#: src/config.rs:896
+msgid "option '{}' does not take a value"
+msgstr "opțiunea '{}' nu necesită(primește) o valoare"
+
+#: src/config.rs:947 src/install.rs:1796
+msgid "There are {n} providers available for {pkg}:"
+msgstr "Există {n} provideri disponibili pentru {pkg}:"
+
+#: src/config.rs:961 src/info.rs:57 src/info.rs:96 src/install.rs:1755
+#: src/install.rs:1804
+msgid "Repository"
+msgstr "Repository"
+
+#: src/config.rs:983
+msgid "  failed to sync {}"
+msgstr " nu s-a putut sincroniza {}"
```

Comment:
Missing a space

Reply:
Saw the typo, fixed now, thanks for pointing it out.